test/unit/alignment/matrix/detail/alignment_trace_matrix_full_banded_test.cpp

```diff
@@ -36,27 +38,30 @@ struct outer_iterator
 template <typename test_type>
 struct iterator_fixture<outer_iterator<test_type>> : alignment_matrix_base_test<test_type>
 {
-    static constexpr trace_directions N = trace_directions::none;
+    static constexpr seqan3::detail::trace_directions N = seqan3::detail::trace_directions::none;
 
     using base_t = alignment_matrix_base_test<test_type>;
     using iterator_tag = std::input_iterator_tag;
     static constexpr bool const_iterable = false;
 
     using base_t::test_range;
-    std::vector<std::pair<std::pair<size_t, size_t>, trace_directions>> expected_range{{{2, 0}, N},
-                                                                                       {{1, 1}, N},
-                                                                                       {{0, 2}, N},
-                                                                                       {{0, 3}, N},
-                                                                                       {{0, 4}, N}};
+    std::vector<std::pair<std::pair<size_t, size_t>,
+                          seqan3::detail::trace_directions>> expected_range{{{2, 0}, N},
+                                                                            {{1, 1}, N},
+                                                                            {{0, 2}, N},
+                                                                            {{0, 3}, N},
+                                                                            {{0, 4}, N}};
 
     template <typename lhs_t, typename rhs_t>
     static void test(lhs_t lhs, rhs_t rhs)
     {
         EXPECT_EQ(lhs.coordinate.second, std::get<0>(std::get<0>(rhs)));
         EXPECT_EQ(lhs.coordinate.first, std::get<1>(std::get<0>(rhs)));
 
-        if constexpr (std::is_same_v<std::tuple_element_t<0, test_type>,
-                                     detail::alignment_trace_matrix_full_banded<trace_directions>>)
+        using full_banded_matrix_with_trace_directions =
+            seqan3::detail::alignment_trace_matrix_full_banded<seqan3::detail::trace_directions>;
+
+        if constexpr (std::is_same_v<std::tuple_element_t<0, test_type>, full_banded_matrix_with_trace_directions>)
```

Comment:
If you only need the type once, I would prefer to put it into the if constexpr
```suggestion
        if constexpr (std::is_same_v<std::tuple_element_t<0, test_type>,
                                     seqan3::detail::alignment_trace_matrix_full_banded<seqan3::detail::trace_directions
                                     >>)
```

or `seqan3::detail::trace_directions>>)` in the last line and intended by 4 spaces.

Reply:
@marehr told me to use 'using' when lines get too long and correct line breaks are unclear.